Vates/VatesAPI/test/vtkDataSetToGeometryTest.h

```diff
@@ -243,10 +236,10 @@ static vtkFieldData* createFieldDataWithCharArray(std::string testData, std::str
     using namespace Mantid::VATES;
 
     using namespace Mantid::VATES;
-    vtkRectilinearGrid* dataA = vtkRectilinearGrid::New();
+    vtkNew<vtkRectilinearGrid> dataA;
```

Comment:
Do I understand correctly that this (and some other equivalent changes) fixes a memory leak?